docs/rackhd/notification.rst

```diff
@@ -0,0 +1,89 @@
+|                 |           | discovery process,it has         | created and all catalog        |
+|                 |           | two cases:                       | jobs are finished              |
+|                 |           |                                  |                                |
+|                 |           | - Automatic discovery            |                                |
+|                 |           | - Passive discovery by           |                                |
+|                 |           |   post a node by REST API        |                                |
++-----------------+-----------+----------------------------------+--------------------------------+
+| removed         | one-shot  | Event occurs when node is        | Triggered when node's all      |
+|                 |           | deleted by REST API              | information are deleted,       |
+|                 |           |                                  | it includes node,catalogs,     |
+|                 |           |                                  | relationships with other       |
+|                 |           |                                  | models, etc.                   |
++-----------------+-----------+----------------------------------+--------------------------------+
+| sku.assigned    | one-shot  | Event occurs when node's `sku`   | Triggered when sku is created  |
+|                 |           | field is assigned.               | and associated with the node   |
++-----------------+-----------+----------------------------------+--------------------------------+
+| sku.unassigned  | one-shot  | Event occurs when node's `sku`   | Triggered when sku is          |
+|                 |           | field is unassigned.             | not associated with the node   |
++-----------------+-----------+----------------------------------+--------------------------------+
+| sku.updateed    | one-shot  | Event occurs when node's `sku`   | Triggered when node's sku      |
+|                 |           | field is updated.                | s associated with another sku  |
++-----------------+-----------+----------------------------------+--------------------------------+
+| obms.assigned   | one-shot  | Event occurs when node's `obms`  | Triggered when obm is created  |
+|                 |           | field is assigned.               | and associated with the node   |
++-----------------+-----------+----------------------------------+--------------------------------+
+| obms.unassigned | one-shot  | Event occurs when node's `obms`  | Triggered when obm is          |
+|                 |           | field is unassigned.             | not associated with the node   |
++-----------------+-----------+----------------------------------+--------------------------------+
+| obms.updateed   | one-shot  | Event occurs when node's `obms`  | Triggered when node don't have |
```

Comment:
Another question, if this event is  triggered when all obm settings are removed, isn't more reasonable to change the `action` to "obm.removed"